### PR TITLE
document repiscomputed

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: src/simplepolyhedron.jl [hunk 1]
```diff
@@ -75,6 +75,11 @@ function Base.push!(p::SimplePolyhedron{N}, ext::VRepresentation{N}) where N
     p.hrep = nothing
 end
 
+"""
+    hrepiscomputed(p::SimplePolyhedron)
```
Comment (blegat) [inline src/simplepolyhedron.jl]: The argument should be `Polyhedron`. Can you also move the doc to the definition in `operations.jl` ?

File: src/simplepolyhedron.jl [hunk 2]
```diff
@@ -86,6 +91,12 @@ function hrep(p::SimplePolyhedron)
     end
     get(p.hrep)
 end
+
+"""
+    vrepiscomputed(p::SimplePolyhedron)
```
Comment (blegat) [inline src/simplepolyhedron.jl]: Same here :)

**Comment by mforets** [comment]
what? i didn't want to close it, just push (-f) because i squashed to my branch. i have the following:

```
$ git remote -v
origin  https://github.com/mforets/Polyhedra.jl.git (fetch)
origin  https://github.com/mforets/Polyhedra.jl.git (push)
upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (fetch)
upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (push)
```

**Comment by mforets** [comment]
ok, sorry, can you reopen my `mforets/doc_repiscomputed` branch, or shall i submit a new PR?

**Comment by blegat** [comment]
Don't know why it closed. I usually also do `push -f` and it does not close the PR

**Comment by mforets** [comment]
> Don't know why it closed. I usually also do push -f and it does not close the PR

i did a `git rebase HEAD~2` and then `git push -f` (i forgot to commit). then it seems like github automatically closes the PR if there are no commits. thanks for the feedback :)

**Pull request merged**